### DISCUSSION TITLE
Show windows from other spaces and/or fullscreen windows

### DISCUSSION THREAD

**Discussion by jameszhan9592** [lawand-dot-io/taskbar#39] [Feature requests]
Currently, only windows from the current space are shown in Taskbar. Some workflows might require seeing windows from other spaces. For example windows that are full-screened get moved into their own space, and thus are no longer listed with the other windows. Also it might be useful to see windows from all spaces in Taskbar.

**Comment by lawand-dot-io** [comment]
@jameszhan9592 I did some research and from what I can see so far, the macOS API doesn't allow applications like Taskbar to access windows from other spaces (including full-screen windows) in a consistent way that always works. I'll keep this in mind as I continue to learn how the macOS API works.

**Reply by JosTheDude** [reply]
uBar appears to do this successfully and Alt+Tab as well -- for reference

**Reply by lawand-dot-io** [reply]
@JosTheDude yeah, I probably figured it out, will test it and if I don't see bugs I'll add it in an update 👍

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/issues/90

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/112

**Comment by jnjmarks** [comment]
Am I interpreting this request properly? Currently, all "pinned apps" in the taskbar (left of the open apps) disappear from their "pinned location" when open and this request is suggesting that their icon remains in the pinned location even when the app is open? Just clarifying because I would love to see this request fulfilled also. 

Personally, I really like seeing what apps I have pinned, regardless of whether they're open or not. 
A suggestion from me: The apps that are pinned remain in the taskbar even when they are open (this allows the user to maintain visual control over their set up) BUT, when the user clicks on a pinned app icon that is already open (i.e. they forgot it was open), the open window glows or highlights for a second to show that it is indeed open...? 

Thanks!

**Reply by lawand-dot-io** [reply]
@jnjmarks What you're mentioning is a different request from this ticket. I have moved it here: https://github.com/lawand-dot-io/taskbar/discussions/140

As always the requests that are most upvoted are more likely to be implemented. Cheers and enjoy Taskbar :)

**Comment by tknman0700** [comment]
I am not seeing icons for apps I have running in full screen. Is there a way to rectify this? Running v.1.3

**Reply by lawand-dot-io** [reply]
hello. what you're mentioning is not a bug. it's a feature not yet implemented in Taskbar..

**Comment by lawand-dot-io** [comment]
Duplicate: https://github.com/lawand-dot-io/taskbar/discussions/188
